backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java

```diff
@@ -105,7 +101,20 @@ public String getFacilityName() {
   }
 
   public DeviceType getDefaultDeviceType() {
-    return this.defaultDeviceSpecimen == null ? null : this.defaultDeviceSpecimen.getDeviceType();
+    return this.defaultDeviceType;
+  }
+
+  public void setDefaultDeviceTypeSpecimenType(DeviceType deviceType, SpecimenType specimenType) {
```

Comment:
thoughts on checking if specimen type is valid for the device type in this method before we set the defaults? that or maybe when creating a new test order

Reply:
fair enough - so the onus is on the caller of the setter to make sure it's passing in valid data, which is the case in the code currently.